app/Traits/CustomFieldsRequestTrait.php

```
@@ -3,12 +3,34 @@
 namespace App\Traits;
 
 use App\Models\CustomField;
+use Illuminate\Support\Facades\Validator;
 
 trait CustomFieldsRequestTrait
 {
 
     public function customFieldRules($rules = [])
     {
+        Validator::extend('valid_multiselect_country', function ($attribute, $value, $parameters, $validator) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)